astropy/modeling/fitting.py

```diff
@@ -1156,6 +1154,62 @@ def _convert_input(x, y, z=None, n_models=1, model_set_axis=0):
     return farg
 
 
+class MinimizeFitter(Fitter):
+    """
+
+    A fitter for the minimize optimizer.
+
+    Parameters
+    ----------
+    method : str or callable(custom)
+        select minimization method
```

Comment:
Could you make it more explicit what methods are supported ? Maybe just refer to the scipy documentation.